src/components/Select.tsx

```diff
@@ -7,40 +9,90 @@ interface SelectProps {
   label?: string;
   options?: Option[];
   value?: number | string;
-  onChange: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  onChange: (value: any) => void;
```

Comment:
These types were a pain - we should really just make every function accept a string and then coerce it to be the type that we need